tests/modules/seq2seq_encoders/pytorch_seq2seq_wrapper_test.py

```diff
@@ -122,3 +122,66 @@ def test_wrapper_raises_if_batch_first_is_false(self):
         with pytest.raises(ConfigurationError):
             lstm = LSTM(bidirectional=True, num_layers=3, input_size=3, hidden_size=7)
             _ = PytorchSeq2SeqWrapper(lstm)
+
```

Comment:
Can you change these tests to check that the state is carried through for different sized batches and just generally change them to test equivalence of actual tensors rather than shapes.